torch/optim/_multi_tensor_functional.py

```diff
@@ -0,0 +1,37 @@
+r"""Functional interface"""
```

Comment:
I am actually curious if this is the best way to split these.
Do you think that having these functional implementations live in adadelta.py would be better? (and `_functional.py` imports it from there as exposes them)?
I might make it easier to find a particular implementation no?

Reply:
I agree that this would make it easier to find the corresponding functions, but it seems that it might cause a circular import. adadelta.py imports _functional.py to call into the functional api in the Adadelta class, if we store the single tensor and multi tensor functionals in adadelta.py we must import .adadelta in _functional.py. 

Alternatively we could call the corresponding functions directly in the adadelta class and not use the functional api in all the optimizer classes, does that make sense?

Reply:
> Alternatively we could call the corresponding functions directly in the adadelta class and not use the functional api in all the optimizer classes, does that make sense?

That sounds good.
I think we want to keep a single functional file to keep an API similar to the one in nn.functional.
But the class version can just call into the local function and the functional simply `from .adadelta import adadelta`.